src/Views/Form/Themes/Sequoia/sections/introduction.php

```diff
@@ -1,17 +1,25 @@
 <?php
+global $post;
 
 // Get headline and description
-$headline    = $this->themeOptions['introduction']['headline'];
-$description = $this->themeOptions['introduction']['description'];
-$image       = $this->themeOptions['introduction']['image'];
+$headline    = ! empty( $this->themeOptions['introduction']['headline'] ) ? $this->themeOptions['introduction']['headline'] : $post->post_title;
```

Comment:
As I mentioned above, I will create and issue to use get template options with default value with`get` function, so we do not have to use `isset` conditions

Reply:
Sounds like a good plan!